blog-service/2023-12-27-index-filter.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Index Filter (Search)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - search
+  - release notes
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We're excited to introduce our new **Index** name filter feature in the search table message footer, which allows you to modify the search query by clicking the index name or view surrounding messages by clicking on the dropdown.<br/><img src={useBaseUrl('img/search/get-started-search/search-page/index-filter.png')} alt="index-filter" width="650" style={{border: '1px solid black'}}/>
```

Comment:
The index field has existed in the system; we have recently started displaying it in the UI along with other metadata values
```
Now, we have included the index field as metadata at the bottom of every message row, along with other metadata. This allows you to modify the search query by clicking the index name or view surrounding messages with the index name by clicking on the dropdown.
```
feel free to updated the text as per your need.

Reply:
Thanks for this info @ssharma-sumo. I will get this updated.